src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -104,7 +104,7 @@ struct TraceExtrinsicCurvature : db::SimpleTag {
 };
 
 /*!
- * \brief The energy density \f$\rho=n_a n_b T^{ab}\f$
+ * \brief The energy density \f$e=n_a n_b T^{ab}\f$
```

Comment:
this change should be in the Energy density tag commit.  Also I see I told you `e` instead of `E` previously; I was incorrect before, `e` is the energy density of a fluid.